src/ert/scheduler/driver.py

```diff
@@ -90,12 +90,16 @@ async def _execute_with_retry(
         error_message: Optional[str] = None
 
         for _ in range(total_attempts):
-            process = await asyncio.create_subprocess_exec(
-                *cmd_with_args,
-                stdin=asyncio.subprocess.PIPE if stdin else None,
-                stdout=asyncio.subprocess.PIPE,
-                stderr=asyncio.subprocess.PIPE,
-            )
+            try:
+                process = await asyncio.create_subprocess_exec(
```

Comment:
It seems that `create_subprocess_exec` will have the same treatment in the driver. Could we make this as an utility function, which we catch `FileNotFoundError` and return `False`?

Reply:
I don't quite understand this. I could create a helper function that returns either a process or False, but I do not see the point of that